pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -440,10 +440,10 @@ func createNodeSecurityGroup(client *gophercloud.ServiceClient, nodeSecurityGrou
 	return nil
 }
 
-func (lbaas *LbaasV2) createLoadBalancer(service *v1.Service, name string, internalAnnotation bool) (*loadbalancers.LoadBalancer, error) {
+func (lbaas *LbaasV2) createLoadBalancer(service *v1.Service, name, clusterName string, internalAnnotation bool) (*loadbalancers.LoadBalancer, error) {
 	createOpts := loadbalancers.CreateOpts{
 		Name:        name,
-		Description: fmt.Sprintf("Kubernetes external service %s", name),
+		Description: fmt.Sprintf("%s external service %s", clusterName, name),
```

Comment:
Can we use that:
Description: fmt.Sprintf("Kubernetes external service %s from cluster %s", name, clusterName),

Reply:
BTW, I want to add 'clusterName' into the description of cloudprovider's resources, such as LoadBalancer, cinder volume, Securty Group, vip, Floatingip ...

Reply:
Just like L1058.